mappings/net/minecraft/util/math/MathUtils.mapping

```diff
@@ -60,14 +66,40 @@ CLASS none/or net/minecraft/util/math/MathUtils
 		ARG 0 values
 	METHOD b fastFloor (D)I
 		ARG 0 value
+	METHOD b modFloor (DD)D
+		ARG 0 num
+		ARG 1 denom
 	METHOD b clampedLerp (DDD)D
+		ARG 0 v0
+		ARG 1 v1
+		ARG 2 time
 	METHOD b cos (F)F
 		ARG 0 value
+	METHOD b modFloor (FF)F
+		ARG 0 num
+		ARG 1 denom
+	METHOD b packRgbFloats (FFF)I
+		ARG 0 r
+		ARG 1 g
+		ARG 2 b
 	METHOD b wrapDegrees (I)I
 		ARG 0 value
-	METHOD b (II)I
+	METHOD b modFloor (II)I
+		ARG 0 num
+		ARG 1 denom
+	METHOD b packRgbInts (III)I
```

Comment:
Both `packRgbFloats` and this could be named `packRgb` - let's make use of overloading!